calcom/cal.com
# Fixed Select component  ':active' option color

## What does this PR do?



I have changed ':active' option color in Select Component, so now it has black background and white color
Fixes #2178

Demo:
https://user-images.githubusercontent.com/16666234/159053955-d2ca8415-5154-4e54-ba19-b3fde5fe1ecd.mp4



## Type of change




## How should this be tested?



Load the app locally, go to Availability tab and check the start and end times of the day Selects, when the option has ':active' state the time is clearly visible

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
